data/Stephens_2023/metadata.yml

```diff
@@ -0,0 +1,106 @@
+source:
+  primary:
+    key: Stephens_2023
+    bibtype: Article
+    year: 2023
+    author: Ruby E. Stephens and Rachael V. Gallagher and Lily Dun and Will Cornwell
+      and Hervé Sauquet
+    title: Insects pollinated flowering plants for most of angiosperm evolutionary
+      history
+    journal: bioRxiv
+    volume: .na
+    number: .na
+    pages: .na
+    doi: 10.1101/2023.01.31.526530
+contributors:
+  data_collectors:
+  - last_name: Stephens
+    given_name: Ruby
+    ORCID: 0000-0002-3767-2690
+    affiliation: Department of Biological Sciences, Macquarie University, Australia
+    additional_role: contact
+  - last_name: Lily
+    given_name: Dun
+    ORCID: 0000-0002-0712-5143
+    affiliation: Evolution & Ecology Research Centre, School of Biological, Earth,
+      and Environmental Sciences, UNSW Sydney, Australia
+  assistants: none
+  austraits_curators: Elizabeth Wenk
+dataset:
+  data_is_long_format: yes
+  custom_R_code: .na
+  collection_date: 2022/2023
+  taxon_name: canonicalName
+  trait_name: trait_name
+  value: value
+  description: Study that models the evolution of pollination across angiosperms and
+    quantify the timing and environmental associations of pollination shifts.
+  basis_of_record: literature
+  life_stage: adult
+  sampling_strategy: Scoring pollination mode ^p Pollination system or syndrome was
```

Comment:
Text for sampling strategy Needs to go in quotes, as text contains a `:`

Reply:
fixed...